test/multidc/bcountermgr_SUITE.erl

```diff
@@ -30,25 +30,24 @@
 
 %% common_test callbacks
 -export([init_per_suite/1,
-         end_per_suite/1,
-         init_per_testcase/2,
-         end_per_testcase/2,
-         all/0]).
+    end_per_suite/1,
+    init_per_testcase/2,
+    end_per_testcase/2,
+    all/0]).
 
 %% tests
--export([
-         test_dec_success/1,
-         test_dec_fail/1,
-         test_dec_multi_success0/1,
-         test_dec_multi_success1/1,
-         conditional_write_test_run/1
-        ]).
+-export([test_dec_success/1,
+    test_dec_fail/1,
+    test_dec_multi_success0/1,
+    test_dec_multi_success1/1,
+    conditional_write_test_run/1,
+    parallel_increment_decrement/1,
+    two_nodes_want_everything/1]).
 
 -include_lib("common_test/include/ct.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
--define(TYPE, antidote_crdt_counter_b).
--define(BUCKET, test_utils:bucket(bcountermgr_bucket)).
+-define(BUCKET, test_utils:bucket(bcountermgr_bucket)). %%TODO Why?
```

Comment:
To ensure that different test cases don't interfere. Resetting the test fixture is too time consuming.

Reply:
Ok I will remove those TODOs